app/api/attendees.py

```diff
@@ -123,6 +130,7 @@ def query(self, view_kwargs):
             query_ = query_.join(User, User.email == TicketHolder.email).filter(User.id == user.id)
 
         query_ = event_query(self, query_, view_kwargs, permission='is_registrar')
+
```

Comment:
Revert

Reply:
But you said it's unnecessary check and will break the end-point ?

Reply:
Revert the newline added